gitops/README.md

```diff
@@ -23,36 +25,112 @@ Post this, a user will be able to log in to kcp workspace and start creating Pip
+  - cluster-setup.yaml
+- credentials
+  - kubeconfig
+    - compute
+    - kcp
+- environment
+  - compute
+```
+
+### Generate kubeconfigs
+
+In order to generate the kubeconfig for the cluster and kcp instance with the appropriate permissions using service accounts, we have created a few scripts which are available [here](../images/access_setup/content/bin). 
+The [README.md](../images/access_setup/README.md) describes the steps but essentially, you need the below two commands to generate kubeconfig for your cluster and kcp instance respectively. Please refer to the Readme for more details.
+
+```
+- ./setup_compute.sh --kubeconfig /home/.kube/mycluster.kubeconfig --work_dir /repo/sre
+- ./setup_kcp.sh --kubeconfig /kcp/kcpinstance.kubeconfig --kcp_workspace plnsvc-ws --work_dir /repo/sre
+```
+
+### Run manual scripts
+
+Currently, we support two ways to get the cluster set up and ready to start creating PipelineRuns, one of which is to run the below scripts manually.  
+_Note: Please run these scripts only if you intend not to use PaC to automate the workflow._
+
+    a. To install Tekton components
+
+    ```
+    $ WORKSPACE_DIR=/home/workspace/pipelines-service/gitops/sre ../images/cluster-setup/install.sh
     ```
 
     b. To access and setup kcp
 
-    ```bash
-    KCP_ORG="root:pipelines-service" KCP_WORKSPACE="compute" DATA_DIR="/home/workspace/pipelines-service" ../images/kcp-registrar/register.sh
     ```
+    $ KCP_ORG="root:pipelines-service" KCP_WORKSPACE="compute" DATA_DIR="/home/workspace/pipelines-service/gitops/sre" ../images/kcp-registrar/register.sh
+    ```
+
+### Install and setup Pipelines As Code (PaC)
+
+The second method which we support is to automate the process by using Pipelines as Code.
 
-2. Open and merge a Pull Request
 There are a couple of steps that need to be done to use PAC in Pipelines-service.
 a. Install Pipelines as Code on the OpenShift Cluster.
-b. Set up a Git application on any of the Git providers (GitHub, GitLab, Bitbucket etc.) by following the steps from [here](https://pipelinesascode.com).
+- Following the instructions from [here](https://pipelinesascode.com/docs/install/installation/) for a OpenShift cluster:
+  ```
+  kubectl patch tektonconfig config --type="merge" -p '{"spec": {"addon":{"enablePipelinesAsCode": false}}}'
+  kubectl apply -f https://raw.githubusercontent.com/openshift-pipelines/pipelines-as-code/stable/release.yaml
+  ```
+b. In order to set up PaC to listen to PRs, you must push the repository created in step 1 to a Git provider of your choice.
+
+c. From the path of your repo in your system, set up a Git application on any of the Git providers (GitHub, GitLab, Bitbucket etc.) by following the steps from [here](https://pipelinesascode.com).
+- As an example, if you wish to set up a GitHub app, we recommend using the [tkn pac cli tool](https://pipelinesascode.com/docs/guide/cli/) as it is easy to complete the whole process in just a couple of commands.
```

Comment:
is it "we" or the PaC authors making this recommendation?
This is an imperative approach, which makes things easy but not declarative, hence not GitOps friendly